docs/guides/commands.md

```diff
@@ -220,3 +220,138 @@ Invokes interactive prompt to remove Volumes from the function configuration
 ```console
 func config volumes remove [-p <path>]
 ```
+
+## `repository`
```

Comment:
Wouldn't it make more sense to move this under `config` instead of a top level command?

Reply:
`config` is specifically managing the config file, and is scoped to a single function.  The `repository` command is managing global settings, and is scoped to the system.
We could of course change this, but that would require a restructuring of config in addition to the creation of the `repository` command.
I am taking, as my reference, the venerable `git` command, and in particular `git remote`.  It  allows for mutation of the `gitconfig` using `git config`, and even a `--global` but is specific to values which appear in the gitconfig, which is similar to our func.yaml.

Reply:
This is probably a good time to mention that I do think we need a `func config --global` flag which mutates a ~/.config/func/config to provide global overrides to default values.